lib/Clam/CfgBuilder.cc

```diff
@@ -2486,6 +2558,24 @@ void CrabIntraBlockBuilder::visitStoreInst(StoreInst &I) {
         m_bb.store_to_ref(ptr->getVar(), m_lfac.mkRegionVar(rgn), typed_const);
       }
     }
+
```

Comment:
The assertion should be added before the translation of store. Unlike, the other instructions where the order in which the fat instrumentation is added is not relevant, for load/store we want to do the check before the actual dereference happens.